src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_factory.py

```diff
@@ -49,7 +48,7 @@ def generic_data_service_factory(service, name=None, key=None, connection_string
 
 
 def storage_client_factory(**_):
-    return get_mgmt_service_client(StorageManagementClient)
+    return get_mgmt_service_client(ResourceType.MGMT_STORAGE_STORAGE_ACCOUNTS)
```

Comment:
What's the reason for this weird format?  There are no storage account dataplane commands.